### DISCUSSION TITLE
Some suggestions

### DISCUSSION THREAD

**Discussion by strongbow25** [Eyal-WowHub/WowInfo#8] [General]
Hi,

Absolutely love this addon. A definite time-saver for info. Just a cfew of suggestions.

- the ability to toggle on and off some of the summaries. For example, in my UI I don't need the Backpack Summary. Also, the Reputation Summary is duplicating info I'm already getting a summary of.
- the ability to move the summaries. In my UI the Game Menu summary is attached to the tooltip location, I'd rather it be like the other Micro Menu summaries and be attached to it's icon.
- probably outside the scope for the addon but support for the other 'reports' (Garrison, Mission Table, etc.)
- and, finally, ElvUI/data bar support would be awesome. For example, hovering over the ElvUI experience, reputation, and honor data bars does not show any summaries.

Thanks again for this awesome add-on and your hard work. :D

**Comment by iam3yal** [comment]
@strongbow25 Thank you for the feedback! 

> the ability to toggle on and off some of the summaries. For example, in my UI I don't need the Backpack Summary. Also, the Reputation Summary is duplicating info I'm already getting a summary of.

No ETA but this is top priority on my todo list!

**Reputation summary should be off by default for all factions unless you explicitly set to track a faction from the options.** but then you might be confusing reputation with currency.

> the ability to move the summaries. In my UI the Game Menu summary is attached to the tooltip location, I'd rather it be like the other Micro Menu summaries and be attached to it's icon.

Same as above. 😄 

> probably outside the scope for the addon but support for the other 'reports' (Garrison, Mission Table, etc.)

I'd love to have more information available but everything takes time so I have to prioritize features but I'll definitely consider it.

> and, finally, ElvUI/data bar support would be awesome. For example, hovering over the ElvUI experience, reputation, and honor data bars does not show any summaries.

The issue with this is maintenance, I don't use ElvUI myself so it's a bit hard to support, if anything I'd offer APIs so people could write their own plugins and add the same information with ease to anywhere.

**Comment by strongbow25** [comment]
"Reputation summary should be off by default for all factions unless you explicitly set to track a faction from the options. but then you might be confusing reputation with currency."

Uhm .. ya. oops. Meant currency. :)

**Comment by iam3yal** [comment]
@strongbow25 Next version will allow you to disable the tooltips you don't need like the `Money` tooltip on the `Backpack` and the `CurrencyTracker` tooltip on currencies in the `Currency` tab.

> In my UI the Game Menu summary is attached to the tooltip location

I've written a personal addon that you can download from [here](https://github.com/Eyal-WowHub/WowFrames) that **should** fix it, the specific code is this:
```lua
local GetMouseFoci = GetMouseFoci

hooksecurefunc("GameTooltip_SetDefaultAnchor", function(tooltip, parent)
    local frame = GetMouseFoci()[1]
    -- Fixes the game menu tooltip.
    if frame == MainMenuMicroButton then
        tooltip:SetOwner(MainMenuMicroButton, "ANCHOR_TOP")
        return
    end
    -- Fixes the experience bar tooltip.
    if frame == MainStatusTrackingBarContainer.bars[4] then
        tooltip:SetOwner(MainStatusTrackingBarContainer.bars[4], "ANCHOR_CURSOR")
        return
    end
end)
```
My addon does few more things so if you want just to adjust the tooltip for the game menu then you can use [Addon Creator](https://addon.bool.no/) and paste the above code there.
